README.md

```diff
@@ -43,22 +43,22 @@ hub credibility.
 
 These and prior disagreements have now made clear the need for an alternative
 hub with a renewed focus and Alignment to serve as the canonical minimal
-IBC/ICS token hub with respect to Cosmos to champion the ideals of sovereignty,
+IBC/ICS token hub for Cosmos to champion the ideals of sovereignty,
 security, and decentralization everywhere; and secondarily to serve as the main
 base for a political party and more-intelligent voting bloc with respect to
```

Comment:
the intent was to mirror "with respect to" here, so the structure is clear for the sentence.

Reply:
The intention with the usage of "for" is to imply that ATONE is the Cosmos hub.  But I get your point.